docs/apache-airflow-providers-databricks/connections/databricks.rst

```diff
@@ -70,6 +70,10 @@ Extra (optional)
 
     * ``token``: Specify PAT to use. Consider to switch to specification of PAT in the Password field as it's more secure.
 
+    Following parameters are necessary if using Service Principal with Oauth token:
```

Comment:
`Oauth` -> `OAuth`. Also would be useful to add a link to the docs: https://docs.databricks.com/en/dev-tools/authentication-oauth.html. Maybe also highlight that this is for AWS only.

Reply:
Fully agree.  Actually I missed adding entries in the `Login` and `Password` sections above too.  I will write them in today.

Reply:
Doc's have been updated now @alexott.  Hope it's ok